src/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+require_relative './rental'
+
+class Book
+  attr_accessor :date, :title, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = nil
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, person, self)
```

Comment:
- [ ] Kindly notice that you have swapped the attributes that you should send to creat the rental.

```ruby
Rental.new(date, self, person)
```